db/migrate/20160115142023_remove_replicated_rows_from_newly_excluded_tables.rb

```diff
@@ -0,0 +1,25 @@
+class RemoveReplicatedRowsFromNewlyExcludedTables < ActiveRecord::Migration
+  class MiqEventDefinition < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+  end
+
+  class ScanItem < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+  end
+
+  class Configuration < ActiveRecord::Base
+    serialize :settings, Hash
+    self.inheritance_column = :_type_disabled
+  end
+
+  def up
+    region_cond = ActiveRecord::Base.region_to_conditions(File.read(File.join(Rails.root, "REGION")).to_i)
+    MiqEventDefinition.where.not(region_cond).delete_all
+    ScanItem.where.not(region_cond).delete_all
+
+    Configuration.where(:typ => "vmdb").each do |c|
```

Comment:
We may have to deal with double serialized configs here...it's a weird old bug...not sure it was ever fixed, and thus we may just need some "just in case" code for it.